tests/unit/api/maintenance-pool.py

```diff
@@ -0,0 +1,98 @@
+"""Tests related to the maintenance pool."""
+from hil import model, config, api
+from hil.test_common import config_testsuite, config_merge, fresh_database, \
+    fail_on_log_warnings, additional_db, with_request_context, \
+    server_init, LoggedWarningError
+from hil.auth import get_auth_backend
+import pytest
+
+OBM_TYPE_MOCK = 'http://schema.massopencloud.org/haas/v0/obm/mock'
+
+
+@pytest.fixture
+def configure():
+    """Configure HIL"""
+    config_testsuite()
+    config_merge({
+        'auth': {
+            'require_authentication': 'False',
+        },
+        'extensions': {
+            'hil.ext.auth.null': None,
+            'hil.ext.auth.mock': '',
+            'hil.ext.switches.mock': '',
+            'hil.ext.obm.ipmi': '',
+            'hil.ext.obm.mock': '',
+            'hil.ext.network_allocators.null': None,
+            'hil.ext.network_allocators.vlan_pool': '',
+        },
+        'hil.ext.network_allocators.vlan_pool': {
+            'vlans': '40-80',
+        },
+        'maintenance': {
+            'maintenance_project': 'maintenance',
+            # Keystone url acts as dummy for posting
+            'url': 'http://127.0.0.1:35357/v3/'
```

Comment:
I would just pick something arbitrary; there' s no reason to confuse this with keystone.